packages/popup/src/pages/CurrentSeed/index.js

```diff
@@ -3,21 +3,27 @@ import Input from '@cfxjs/component-input'
 import Button from '@cfxjs/component-button'
 import {Selected} from '@cfxjs/component-icons'
 import {CompWithLabel} from '../../components'
-import create from '../../hooks/zustand.js'
+import create from '../../hooks/zustand'
 
 const useStore = create(
   (set, get) => ({
     // value
     hdGroup: [],
     creatingAccount: false,
-    accountCreatationError: null,
+    accountCreationError: null,
     accountName: '',
     selectedGroup: null,
 
     // hook
     groupAfterSet: ({groupData}) => {
-      set({hdGroup: groupData.filter(({vault: {type}}) => type === 'hd')})
-      if (!get().selectedGroup) set({selectedGroup: get().group.groupData[0]})
+      const hdGroup = groupData.filter(g => g?.vault?.type === 'hd')
+      set({hdGroup})
+      if (!get().selectedGroup) set({selectedGroup: hdGroup[0]})
```

Comment:
i think things still might going wrong here if we only set selectedGroup if it's falsy

Reply:
done